.github/workflows/ci.yaml

```diff
@@ -686,7 +686,7 @@ jobs:
       - name: Run pytest
         run: |
           . venv/bin/activate
-          python3 -X dev -bb -m pytest \
+          python3 -X dev -b -m pytest \
```

Comment:
This is working around an issue, with the side-effect of affecting ALL tests.

IMHO, this is not something we should do.

Reply:
I agree it should not be lumped in with a simple PR to bump a dependency, so we should consider the changing of the option as a separate PR. I can open that PR, but do you have any opposition to it out of hand?

Reply:
Well changing this option will just generate a lot of warnings and allow for things we have blocked and don't want, instead of erroring out.

So, IMHO, this should not be changed.

Reply:
The option was only added this past month without a clear idea of what it does. https://github.com/home-assistant/core/pull/58079
It seems like the option is mainly for catching errors when migrating from python2 to python3, so doesn't seem very critical for us.

Reply:
@scop @MartinHjelmare Any thoughts on this?

Reply:
I want to add: If we do change this; We should fully remove `-bb` (and not use `-b`). If we don't care enough to make it an error, then lets at least not flood the logs with warnings.

Reply:
Opened a PR to remove `-bb` here: https://github.com/home-assistant/core/pull/59846